thrift0.13/src/main/java/com/linecorp/armeria/common/thrift/ThriftSerializationFormats.java

```diff
@@ -55,22 +49,5 @@
      */
     public static final SerializationFormat TEXT_NAMED_ENUM = SerializationFormat.of("ttext-named-enum");
 
-    private static final Set<SerializationFormat> THRIFT_FORMATS =
-            ImmutableSet.of(BINARY, COMPACT, JSON, TEXT, TEXT_NAMED_ENUM);
-
-    /**
-     * Returns the set of all known Thrift serialization formats.
-     */
-    public static Set<SerializationFormat> values() {
-        return THRIFT_FORMATS;
-    }
-
-    /**
-     * Returns whether the specified {@link SerializationFormat} is Thrift.
-     */
-    public static boolean isThrift(SerializationFormat format) {
-        return values().contains(requireNonNull(format, "format"));
-    }
-
```

Comment:
Could we revive these methods? `values()` could pull the values from `getThriftSerializationFormats()` you added.

Reply:
What you suggest is to have the link `SerializationFormat` -> `TProtocolFactory` in this class rather than `ThriftProtocolFactories` right? We certainly don't need to access it from both

Reply:
Understood your intent and fixed. 👍